spec/features/events_spec.rb

```diff
@@ -34,7 +34,28 @@
       visit event_path(@event)
       expect(page).to have_text(I18n.t "free_places", count: (@event.max_participants).to_i - (i+1), scope: [:events, :applicants_overview])
       expect(page).to have_text(I18n.t "occupied_places", count: (i+1), scope: [:events, :applicants_overview])
-    end 
+    end
+  end
+
+  scenario "logged in as Organizer I can change application status with radio buttons" do
+    login(:organizer)
+
+    @pupil = FactoryGirl.create(:profile)
+    @application_letter = FactoryGirl.create(:application_letter, event: @event, user: @pupil.user)
+    visit event_path(@event)
+    ApplicationLetter.statuses.keys.each do |new_status|
+      choose(I18n.t "application_status.#{new_status}")
+      ApplicationLetter.where(id: @application_letter.id, status: new_status)
```

Comment:
are you actually testing, that the status has changed? I don't see that here as you don't do anything with the return value of that request?